ui/app/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -102,7 +102,7 @@ export default function AwaitingSwap({
       request_type: fetchParams?.balanceError ? 'Quote' : 'Order',
       slippage: fetchParams?.slippage,
       custom_slippage: fetchParams?.slippage === 2,
-      gas_fees: feeinFiat,
+      gas_fees: feeinUnformattedFiat,
```

Comment:
do we also want to trade the usdConversionRate *at the time of transaction* or no?

Reply:
yeah, sounds like a good idea

Reply:
oh, actually, maybe not...

Reply:
I don't think this gives us anything we need. We might want to also add the used gas limit here (as we already know the price). But I think that is for a separate PR.

Reply:
cool, thanks @danjm